examples/src/components/MeasureValueFilterExample.jsx

```diff
@@ -1,24 +1,24 @@
-// (C) 2007-2019 GoodData Corporation
+// (C) 2007-2020 GoodData Corporation
 import React, { Component } from "react";
 import { PivotTable, Model } from "@gooddata/react-components";
 
 import "@gooddata/react-components/styles/css/main.css";
-import { projectId, franchiseFeesIdentifier, locationNameDisplayFormIdentifier } from "../utils/fixtures";
+import { projectId, locationNameDisplayFormIdentifier, franchisedSalesIdentifier } from "../utils/fixtures";
```

Comment:
changed from `franchiseFees` to `franchisedSales` because the formaer was not working.